generator/src/pkg/generate/generate.go

```diff
@@ -38,83 +37,102 @@ const (
 	redisImageURL  = "k8s.gcr.io/redis:e2e"
 
 	fortioImageName = "fortio"
-	fortioImageUrl = "fortio/fortio"
+	fortioImageUrl  = "fortio/fortio"
 
 	workerImageName = "worker"
 	workerImageURL  = "redis-demo-worker"
 
-	protocol      = "http"
-	redisProtocol = "tcp"
-	fortioProtocol = "tcp-fortio"
+	protocol         = "http"
+	redisProtocol    = "tcp"
+	fortioProtocol   = "tcp-fortio"
 	fortioUiProtocol = "http-fortio-ui"
 
-	namespace = "edge-namespace"
```

Comment:
I imagine this was removed since we plan to not hard-code one namespace but support several, right?

Reply:
Yes, We get the namespace from the user.